actors/user/src/main/java/org/sunbird/user/dao/impl/UserDaoImpl.java

```diff
@@ -29,8 +29,7 @@
   private CassandraOperation cassandraOperation = ServiceFactory.getInstance();
   private ObjectMapper mapper = new ObjectMapper();
   private static UserDao userDao = null;
-  private static final String KEYSPACE_NAME = "sunbird";
-  private static final String USER_TABLE_NAME = "user";
+  private final Util.DbInfo userDbInfo = Util.dbInfoMap.get(JsonKey.USER_DB);
```

Comment:
Suggestion is to move away from maintaining a static list of db info map in memory. The table name can be defined locally as a constant in this class itself. The keyspace name be a global constant shared by all Dao classes.